int32 botID
int32 deliveryLocationX
int32 deliveryLocationY
---
bool success